build/release/script/dev_kyuubi_vote.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -o pipefail
+set -e
+
+RELEASE_DIR="$(cd "$(dirname "$0")"/..; pwd)"
+
+######### Please modify the variables ##########
+# release version, e.g. 1.4.0-incubating
+release_version=${release_version-""}
```

Comment:
`RELEASE_VERSION=${RELEASE_VERSION:?"RELEASE_VERSION is required"}`

Reply:
thanks, fixed